gensim/test/test_similarities.py

```diff
@@ -538,12 +538,16 @@ def testWord2Vec(self):
         self.assertLoadedIndexEqual(index, model)
 
     def testFastText(self):
-        ft_home = os.environ.get('FT_HOME', None)
-        ft_path = os.path.join(ft_home, 'fasttext') if ft_home else None
-        if not ft_path:
-            return
-        corpus_file = datapath('lee.cor')
-        model = fasttext.FastText.train(ft_path, corpus_file)
+        class LeeReader(object):
+            def __init__(self, fn):
+                self.fn = fn
+
+            def __iter__(self):
+                with smart_open(self.fn, 'r', encoding="cp1251") as infile:
```

Comment:
This doesn't look correct. Why do you think the encoding is `cp1251`?

Reply:
Because UTF-8 doesn't work here, I pick `cp1251` randomly and this works fine.

Reply:
Pick randomly? O_o

No, that encoding doesn't seem correct.

Reply:
```bash
ivan@P50:~/release/gensim$ file -i gensim/test/test_data/lee.cor 
gensim/test/test_data/lee.cor: text/plain; charset=iso-8859-1
```
[iso-8859-1](https://en.wikipedia.org/wiki/ISO/IEC_8859-1), although `cp1251` may not be correct, but it works quite correctly, surprisingly.

Reply:
Not so surprising, both are 1 byte encodings. Please fix the encoding before we forget.

Reply:
Done: https://github.com/RaRe-Technologies/gensim/pull/1931